linkfile.go

```diff
@@ -18,6 +19,14 @@ func NewLinkFile(target string, stat os.FileInfo) File {
 	return lf
 }
 
+func (lf *Symlink) Mode() os.FileMode {
+	panic("Implement me")
```

Comment:
reason to panic rather than return the unix epoch / default timestamp?

Reply:
Only filesystem files and webfiles have an initial implementation at this stage, support for symbolic links, directories, multipart files, etc. remains WIP and therefore I want them to fail loudly as a reminder.

I've hopefully further clarified this in the initial PR comment.

Thanks for taking a look, let me know if you see anything else of concern.